packages/ember-htmlbars/tests/helpers/unbound_test.js

```diff
@@ -313,7 +313,7 @@ QUnit.module('ember-htmlbars: {{#unbound boundHelper arg1 arg2... argN}} form: r
 });
 
 QUnit.test('should be able to render an unbound helper invocation', function() {
-  registry.register('helper:-repeat', helper(function([value], {count}) {
+  registry.register('helper:-repeat', helper(function([value], { count }) {
```

Comment:
This was caught due to bug fixes in JSCS 2.x (it would previously skip this section because it couldn't parse).